doc-surrealdb_versioned_docs/version-1.x/faqs/known-issues.mdx

```diff
@@ -0,0 +1,24 @@
+---
+sidebar_position: 2
+sidebar_label: Known Issues
+title: Known Issues
+description: The purpose of this document is to hightlight some of the known issues with SurrealDB. This will help you to understand the current limitations and issues that are being worked on.
```

Comment:
hightlight -> highlight (typo)

How about something like this?

highlight some of the known issues with SurrealDB, with workarounds if available.